src/app/home-page/widget/entry-box/entry-box.component.html

```diff
@@ -3,7 +3,9 @@
     <div id="workflow-number" fxLayout="row">
       <mat-card-header fxLayout="row">
         <mat-card-title>{{ entryTypeLowerCase | titlecase }}s</mat-card-title>
-        <span class="bubble">{{ totalEntries }}</span>
+        <span class="bubble" id="bubble-green" *ngIf="entryTypeLowerCase === 'workflow'">{{ totalEntries }}</span>
```

Comment:
I suggest doing the styling based on a class, not the id. It just feels better :) Plus there's a chance you could end up with duplicate ids; there are several instances of the entry-box.component -- each entry-box.component should have different a entry type, and hence a unique id, but it's not enforced and you never know if another instance might be instantiated.

Also, I suggest changing `entryTypeLowerCase === 'workflow'` to (untested) `entryType === entryTypeEnum.WORKFLOW` (you'll need to add a property in the TS file for this to work) -- relies on types the compiler can catch instead of spelling a string correctly.

Reply:
Done! For the ids, they were replaced by the preexisting global styles in `styles.scss` following Kathy's comment.